test/app_deployment.yml.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+. ./store.sh
+
+set_val app.CONJUR_AUTHN_URL "$(get_val CONJUR_URL)/authn-k8s/$(get_val AUTHENTICATOR_ID)"
+set_val app.CONJUR_APPLIANCE_URL "$(get_val CONJUR_URL)"
+set_val app.CONJUR_ACCOUNT "$(get_val CONJUR_ACCOUNT)"
+set_val app.CONJUR_AUTHN_LOGIN "host/conjur/authn-k8s/$(get_val AUTHENTICATOR_ID)/apps/$(get_val TEST_NAMESPACE)/service_account/$(get_val APP_SERVICE_ACCOUNT)"
+set_val app.CONJUR_SSL_CERTIFICATE_SECRET "$(get_val CONJUR_RELEASE_NAME)-conjur-ssl-cert"
+set_val app.APP_SERVICE_ACCOUNT "$(get_val APP_SERVICE_ACCOUNT)"
+set_val app.CONJUR_KUBERNETES_AUTHENTICATOR_RELEASE_VERSION "$(get_val CONJUR_KUBERNETES_AUTHENTICATOR_RELEASE_VERSION)"
+
+echo "
```

Comment:
We should definitely try to surface any errors/failures early on. @sgnn7 Did you have any fragile failures specific to this in mind ? It's not clear to me yet how this would be any more fragile than any alternatives.